R/driver_rds.R

```diff
@@ -157,6 +175,14 @@ R6_driver_rds <- R6::R6Class(
         write_if_missing("TRUE", driver_rds_config_file(path, "mangle_key_pad"))
         write_if_missing("TRUE", driver_rds_config_file(path, "compress"))
         write_if_missing("md5", driver_rds_config_file(path, "hash_algorithm"))
+        write_if_missing(
+          deparse(mangle_key_encode),
```

Comment:
One downside of this approach is that the `deparse` route is not going to result in something that can be parsed back out as a usable function later